lib/extractComponentOptions.js

```diff
@@ -0,0 +1,51 @@
+const { readFileSync } = require('fs')
+
+const acorn = require('acorn')
+const dynamicImport = require('acorn-dynamic-import')
+const inject = require('acorn-dynamic-import/lib/walk')
+const walker = inject.default(require('acorn-walk'))
+// Must not be an explicit dependency to avoid version mismatch issue.
+// See https://github.com/nuxt-community/nuxt-i18n/issues/297
+const compiler = require('vue-template-compiler')
+// const { COMPONENT_OPTIONS_BLOCK, COMPONENT_OPTIONS_KEY } = require('constants')
+
+function extractComponentOptions (path, blockName, key) {
```

Comment:
I think you should rename the function and file from `extractComponentOptions`  to  `extractPageOptions`, because it is relative to the `*.vue` file in the `Pages` folder.